avocado/utils/asset.py

```diff
@@ -285,20 +284,17 @@ def _verify(self, asset_path):
         value as the hash of the asset_file, otherwise return False.
         :rtype: bool
         """
-        if not self.asset_hash:
```

Comment:
This test should be `if self.asset_hash is None`. I think it was like that... not sure why it changed.

Reply:
Okay, I'll revert on v2.